runtime/lua/vim/ui.lua

```diff
@@ -141,7 +141,9 @@ function M.open(path)
 
   local cmd --- @type string[]
 
-  if vim.fn.has('mac') == 1 then
+  if vim.fn.executable(vim.g.netrw_browsex_viewer) == 1 then
```

Comment:
`vim.fn.executable()` will error with a `nil` argument.

```suggestion
  if vim.g.netrw_browsex_viewer and vim.fn.executable(vim.g.netrw_browsex_viewer) == 1 then
```